### PR TITLE
docs: add vitepress setup

### PR DESCRIPTION
**Pull request by mkilpatrick**

### CODE REVIEW

**Review by benlife5 — APPROVED** [review]

**Review by briantstephan — APPROVED** [review]

File: packages/pages/docs/.vitepress/config.ts [hunk 1]
```diff
@@ -1,33 +1,73 @@
...
-    {
-      text: "API",
-      items: [{ text: "API Docs", link: "/api/index.html" }],
+    sidebar: {
+      "/guide/": { base: "/guide/", items: sidebarGuide() },
+      "/api/": { base: "/api/", items: [{}] },
     },
-  ];
-}
+    // {
```
Comment (asanehisa) [inline packages/pages/docs/.vitepress/config.ts:66]: is this commented out code purposefully left ?

File: packages/pages/docs/guide/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+If you set up a remote GitHub repository for your project following the `yext pages new` command, your project should be living under your GitHub account. If you have not done this step, you should set up a remote repo on your GitHub account before deploying.
+
+Once your Pages code is in a GitHub repository, you can connect your repo to your Yext account by following the directions (ADD LINK HERE)here(ADD LINK HERE).
+
+By default, any new pushes to your GitHub repo will trigger a re-deploy of your Yext account.
+
+## (Legacy) Projects Using < PagesJS 1.0.0
+
+If your project leverages a PagesJS version less than v1.0.0, you are likely using a version of a starter that was configured using our legacy `package.json` commands (refer to (ADD LINK HERE)this commit(ADD LINK HERE) as an example). You should check the commands to confirm, but the old dev commands will likely be as follows:
```
Comment (asanehisa) [inline packages/pages/docs/guide/getting-started.md:71]: did we wanna do the "ADD LINK HERE" in this CR?

File: packages/pages/docs/.vitepress/config.ts (hunk 1 above)
Comment (mkilpatrick) [reply packages/pages/docs/.vitepress/config.ts:66]: Yea, I'll be updating it (or removing it) later.

File: packages/pages/docs/guide/getting-started.md (hunk 2 above)
Comment (mkilpatrick) [reply packages/pages/docs/guide/getting-started.md:71]: That will be in a followup. This PR was to get the main functionality working and not focusing on the docs themselves. I just pulled this from HH and removed the breaking link tags they had.

**Review by asanehisa — APPROVED** [review]

**Pull request merged**